index.md

```diff
@@ -1,15 +1,20 @@
-# Welcome to Intent Architect documentation.
+# Intent Architect Documentation
 
-## Where would you like to start ?
+We've organized the documentation in such a way that you would find what you're looking for in no time.
 
-### [Overview of Intent Architect](articles/overview/intro.md)
-An overview of Intent Architect, how it works, and why it provides a better way to build software.
+## [Overview of Intent Architect](articles/overview/index.md)
+An overview of Intent Architect. Why Intent Architect? How does it work? Isn't Code Generation evil? Those questions are answered here.
```

Comment:
Sorry, I know I seem like a bit of a killjoy here 😛. Even if some people do, I don't think we should be announcing that they think code generation is evil. It may get people thinking in the wrong way about or distracted from our offering. especially newer devs which may not even be aware of this opinion.

We could have an article somewhere to the effect of `Code Generation done right in Intent Architect`, or something else like that to address concerns people have had with code gen in the past.

```suggestion
An overview of Intent Architect, how it works, and why it provides a better way to build software.
```

Reply:
I think I found that description in the FAQ and must have accidentally added it there. Happy to change.